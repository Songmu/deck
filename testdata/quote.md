# Title with <q>quoted text</q>

## Subtitle with <q>a quote</q> example

Some text with <q>inline quote</q> in the middle.

<q class="custom">Custom styled quote</q> with a class.

---

# Various quote styles

- Regular quote: <q>This is a quote</q>
- **Bold with quote**: **<q>Bold quoted text</q>**
- *Italic with quote*: *<q>Italic quoted text</q>*
- Quote with `code`: <q>This is `inline code` in a quote</q>
- Nested formatting: <q>Quote with **bold** and *italic*</q>