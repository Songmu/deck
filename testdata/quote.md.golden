[
  {
    "layout": "",
    "titles": [
      "Title with \u003cq\u003equoted text\u003c/q\u003e"
    ],
    "subtitles": [
      "Subtitle with \u003cq\u003ea quote\u003c/q\u003e example"
    ],
    "bodies": [
      {
        "paragraphs": [
          {
            "fragments": [
              {
                "value": "Some text with "
              },
              {
                "value": "inline quote",
                "quote": true
              },
              {
                "value": " in the middle."
              }
            ]
          },
          {
            "fragments": [
              {
                "value": "Custom styled quote",
                "quote": true,
                "className": "custom"
              },
              {
                "value": " with a class."
              }
            ]
          }
        ]
      }
    ]
  },
  {
    "layout": "",
    "titles": [
      "Various quote styles"
    ],
    "bodies": [
      {
        "paragraphs": [
          {
            "fragments": [
              {
                "value": "Regular quote: "
              },
              {
                "value": "This is a quote",
                "quote": true
              }
            ],
            "bullet": "-"
          },
          {
            "fragments": [
              {
                "value": "Bold with quote",
                "bold": true
              },
              {
                "value": ": "
              },
              {
                "value": "Bold quoted text",
                "bold": true,
                "quote": true
              }
            ],
            "bullet": "-"
          },
          {
            "fragments": [
              {
                "value": "Italic with quote",
                "italic": true
              },
              {
                "value": ": "
              },
              {
                "value": "Italic quoted text",
                "italic": true,
                "quote": true
              }
            ],
            "bullet": "-"
          },
          {
            "fragments": [
              {
                "value": "Quote with "
              },
              {
                "value": "code",
                "code": true
              },
              {
                "value": ": "
              },
              {
                "value": "This is ",
                "quote": true
              },
              {
                "value": "inline code",
                "code": true,
                "quote": true
              },
              {
                "value": " in a quote",
                "quote": true
              }
            ],
            "bullet": "-"
          },
          {
            "fragments": [
              {
                "value": "Nested formatting: "
              },
              {
                "value": "Quote with ",
                "quote": true
              },
              {
                "value": "bold",
                "bold": true,
                "quote": true
              },
              {
                "value": " and ",
                "quote": true
              },
              {
                "value": "italic",
                "italic": true,
                "quote": true
              }
            ],
            "bullet": "-"
          }
        ]
      }
    ]
  }
]